storage/pkg/flagset/gateway.go

```diff
@@ -127,81 +135,47 @@ func GatewayWithConfig(cfg *config.Config) []cli.Flag {
 			Name:        "storage-home-provider",
 			Value:       "/home",
 			Usage:       "mount point of the storage provider for user homes in the global namespace",
-			EnvVars:     []string{"STORAGE_STORAGE_HOME_PROVIDER"},
+			EnvVars:     []string{"STORAGE_REGISTRY_HOME_PROVIDER"},
 			Destination: &cfg.Reva.StorageRegistry.HomeProvider,
 		},
 
 		&cli.StringFlag{
-			Name:        "frontend-url",
+			Name:        "public-url",
 			Value:       "https://localhost:9200",
 			Usage:       "URL to use for the storage service",
-			EnvVars:     []string{"STORAGE_FRONTEND_URL"},
-			Destination: &cfg.Reva.Frontend.URL,
+			EnvVars:     []string{"STORAGE_FRONTEND_PUBLIC_URL"},
+			Destination: &cfg.Reva.Frontend.PublicURL,
 		},
 		&cli.StringFlag{
 			Name:        "datagateway-url",
 			Value:       "https://localhost:9200/data",
 			Usage:       "URL to use for the storage datagateway",
-			EnvVars:     []string{"STORAGE_DATAGATEWAY_URL"},
-			Destination: &cfg.Reva.DataGateway.URL,
+			EnvVars:     []string{"STORAGE_DATAGATEWAY_PUBLIC_URL"},
```

Comment:
need to be changed in drone https://github.com/owncloud/ocis/blob/d422a91a01b98893d0a10a7555092342755ee33b/.drone.star#L560

Reply:
Done.